core/src/main/java/com/xatkit/core/recognition/IntentRecognitionProviderFactory.java

```diff
@@ -35,6 +36,20 @@
  */
 public class IntentRecognitionProviderFactory {
 
+    /**
+     * The database model that will be used for this instance of Xatkit.
+     * <p>
+     * This property is optional, and it's default value is set to "MAPDB" if not specified
+     */
+    private static final String XATKIT_DATABASE_KEY = "xatkit.database.model";
+
+    /**
+     * The default value for xatkit.database.model in case it's not specified in the properties file.
+     */
+    private static final String DEFAULT_DATABASE = "MAPDB";
+
+    private static final String INFLUX_DATABASE = "influxDB";
```

Comment:
Renaming it DATABASE_MODEL_INFLUXDB="influxdb".
I put in lower case all the constants, this way we can easily check the value of the properties in a case insensitive way:
```
configuration.getString(DATABASE_MODEL_KEY, DEFAULT_DATABASE_MODEL).toLowerCase().equals(DATABASE_MODEL_INFLUXDB)
```

Reply:
Since now the property is not case sensitive this won't break existing bots using "MAPDB" or "influxDB" 🙂